test/ring/middleware/oauth2_test.clj

```diff
@@ -75,12 +74,12 @@
                          (assoc :session {::oauth2/state "xyzxyz"})
                          (assoc :query-params {"code" "abcabc", "state" "xyzxyz"}))
             response (test-handler request)
-            expires  (-> 3600 time/seconds time/from-now)]
+            expires  (#'oauth2/seconds-from-now-to-date 3600)]
```

Comment:
Accessing a private var here saves a little code, but also ties the test case to the internals of the namespace being tested. Ideally we test only the public API. So instead, just inline the function:

```
            expires  (-> (Instant/now) (.plusSeconds 3600) (Date/from))
```

Reply:
OK, though since identical code is used at four places in the test suite I might as well introduce a helper function for it in the tests I guess?

Reply:
I don't think I mind. You could duplicate `seconds-from-now-to-date` in the test namespace if you wanted.

Reply:
I did it the other way, see what you think.

Reply:
That works, too. Can you ensure that the lines modified in ring.middleware.oauth are within 80 characters?

Reply:
I believe the modified lines are all within 80 characters.

Reply:
I think there's one line in your patch that might be 85?

Reply:
As you perhaps noted the 2nd commit made that like shorter.

Reply:
Oh, weird. Is the GitHub diff incorrect then? In any case, it looks like there are other lines over from previous commits, so I can always fix it with a later formatting commit. I just need the commits to be squashed down and then we're ready to merge.